feat(workflow): Anbox Cloud Demo - add ci workflow for static checks [WD-26312] (#3)
canonical/anbox-cloud-demos: README.md

# Anbox Cloud Demos

> A repo to showcase how android apps can use Anbox Cloud Appliance for local development, and to demo how Anbox can be part of github workflows for running ui screenshot tests and generating test reports.

Reviewer: ```suggestion
A repository to showcase how Android applications can use the Anbox Cloud Appliance for local development, and to demo how it can be used as part of GitHub workflows for running UI screenshot tests and generating test reports.
```
